python/ray/serve/scripts.py

```diff
@@ -371,24 +295,15 @@ def run(
     type=str,
     help=RAY_DASHBOARD_ADDRESS_HELP_STR,
 )
-@click.option(
```

Comment:
Do we want to allow users to get the output in JSON? It can make it easier to process the output programatically. I don't have a strong preference.

If we do get rid of it, we need to update the `serve info` calls in `test_cli` since they print and process JSON.